xml/containers-get.xml

```diff
@@ -0,0 +1,586 @@
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         org.opencontainers.image.<emphasis role="strong">version</emphasis>
+         com.suse.sle.base.<emphasis role="strong">version</emphasis>
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         org.opencontainers.image.version
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         Version of the image
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         &lt;codestream&gt;.&lt;sp&gt;.&lt;cicnt&gt;.&lt;bldcnt&gt; 15.0.4.2
+        </para>
+       </entry>
+      </row>
+      <row>
+       <entry align="left" valign="top">
+        <para>
+         ✓
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         org.opencontainers.image.<emphasis role="strong">created</emphasis>
+         com.suse.sle.base.<emphasis role="strong">created</emphasis>
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         org.opencontainers.image.created
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         Date/time of image build
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         2018-07-27T14:12:30Z
+        </para>
+       </entry>
+      </row>
+      <row>
+       <entry align="left" valign="top">
+        <para>
+         X
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         org.opencontainers.image.<emphasis role="strong">vendor</emphasis>
+         com.suse.sle.base.<emphasis role="strong">vendor</emphasis>
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         org.opencontainers.image.vendor
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         Vendor of the image
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         SUSE LLC
+        </para>
+       </entry>
+      </row>
+      <row>
+       <entry align="left" valign="top">
+        <para>
+         X
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         org.opencontainers.image.<emphasis role="strong">url</emphasis>
+         com.suse.sle.base.<emphasis role="strong">url</emphasis>
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         org.opencontainers.image.url
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         URL with more information
+        </para>
+       </entry>
+       <entry align="left" valign="top">
+        <para>
+         <link xlink:href="https://www.suse.com/products/server/">https://www.suse.com/products/server/</link>
```

Comment:
choose:
* Use self-closing link tag?
* Should this even be linked? Other entries in this column don't appear to be linked.